deploy/charts/alluxio/templates/helpers/_alluxio-site-properties.tpl

```diff
@@ -46,6 +46,9 @@ alluxio.worker.page.store.type=LOCAL
 alluxio.worker.page.store.dirs={{ .Values.pagestore.hostPath }}
 {{ printf "alluxio.worker.page.store.sizes=%v" .Values.pagestore.quota }}
 
+# Worker Identity
+{{ printf "alluxio.worker.identity.uuid.file.path=%v/worker_identity" (include "alluxio.mount.basePath" "/worker_identity") }}
```

Comment:
i thought %v maps to the "alluxio.mount.basePath" what's the second "/worker_identity" following after?